src/collections/views/CollectionList/CollectionList.tsx

```diff
@@ -177,7 +177,7 @@ export const CollectionList: React.FC<CollectionListProps> = ({ params }) => {
         onTabSave={() => openModal("save-search")}
         onTabUpdate={onPresetUpdate}
         tabs={presets.map(tab => tab.name)}
-        loading={loading}
+        loading={!data}
         disabled={loading}
```

Comment:
You can remove `loading` above

```suggestion
        loading={!data}
        disabled={!data}
```